### ISSUE TITLE
Feature request : disabled

### ISSUE THREAD

**Issue by sm2017** [retyui/react-native-confirmation-code-field#35]
disabled props is needed

**Comment by retyui** [comment]
How it will be work?

**Comment by sm2017** [comment]
Line textField , when disabled is true , you cannot modify code

**Comment by retyui** [comment]
```
<CodeInput onFulfill={this.handlerOnFulfill} inputProps={{editable: false}} />
```

**Comment by cjpete** [comment]
@retyui whilst passing editable: false does prevent the cursor from activating, it still seems to clear the code out of the field.

**Comment by retyui** [comment]
@cjpete can you create a video to understand the problem?

**Comment by cjpete** [comment]
> @cjpete can you create a video to understand the problem?

Hopefully this works:
![CodeEditField](https://user-images.githubusercontent.com/13134961/54434850-1437ab00-4727-11e9-8c03-9edb697b1ba2.gif)

**Comment by retyui** [comment]
@cjpete Great! Thank you!

**Comment by retyui** [comment]
@cjpete fixed in 3.2.3
